File: app-service-comments/src/main/java/com/github/arhor/aws/graphql/federation/comments/data/repository/UserRepository.java
package com.github.arhor.aws.graphql.federation.comments.data.repository;

import com.github.arhor.aws.graphql.federation.comments.data.entity.UserEntity;
import org.springframework.data.repository.CrudRepository;

import java.util.UUID;

public interface UserRepository extends CrudRepository<UserEntity, UUID> {
}

File: app-service-posts/src/main/kotlin/com/github/arhor/aws/graphql/federation/posts/data/repository/UserRepository.kt
package com.github.arhor.aws.graphql.federation.posts.data.repository

import com.github.arhor.aws.graphql.federation.posts.data.entity.UserEntity
import org.springframework.data.repository.CrudRepository
import java.util.UUID

interface UserRepository : CrudRepository<UserEntity, UUID>
